strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactResolutionServiceImpl.java

```diff
@@ -119,5 +120,22 @@ public URL resolveArtifactResource(String storageId,
                                    .resolve(artifactResource)
                                    .toURL();
     }
+
+    @Override
+    public RepositoryPath getPath(String storageId,
+                                  String repositoryId,
+                                  String artifactPath) 
+           throws IOException
+    {
+        artifactOperationsValidator.validate(storageId, repositoryId, artifactPath);
```

Comment:
do we really need to validate `getPath` operation?

Reply:
@sbespalov   Just checked it now . I dont think we need it atleast for current implementation . As controller method checks for validity of storageId and repositoryId. Should I remove it ?

Reply:
yes, I think so